dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -90,7 +118,26 @@ public class PatientController : Controller
 
             var responseContent = await ExecuteRestRequest(personId, recordId, uriBuilder.Uri, HttpMethod.Get, "2.0-preview");
 
-            return SafeJsonConvert.DeserializeObject<TimelineResponse>(responseContent, GetDeserializationSettings());
+            return SafeJsonConvert.DeserializeObject<TimelineResponse>(responseContent, _deserializerSettings);
+        }
+
+        private async Task PostTaskTracking(Guid personId, Guid recordId, Guid taskId, DateTimeOffset trackingDateTime)
+        {
+            var restHealthVaultUrl = WebConfigurationManager.AppSettings.Get("HV_RestHealthServiceUrl"); //TODO: use built in SDK function to retreive config settings when available
+
+            // Construct URL
+            var uriBuilder = new UriBuilder(restHealthVaultUrl)
+            {
+                Path = "TaskTracking"
+            };
+
+            var taskTracking = new
+            {
+                taskId,
+                trackingDateTime = trackingDateTime.ToZonedDateTime()
+            };
+
+            await ExecuteRestRequest(personId, recordId, uriBuilder.Uri, HttpMethod.Post, "2.0-preview", SafeJsonConvert.SerializeObject(taskTracking, _serializerSettings));
```

Comment:
Now a constant at the top of the controller, since in this case we're manually overriding what the SDK does